### ISSUE TITLE
Does not show any NL-Alert information anymore

### ISSUE THREAD

**Issue by Ray-Hoo** [glsf91/nlalert#7]
Since a couple of weeks, there is no NL-Alert information shown. It gives an error 404 page not found.

**Comment by glsf91** [comment]
Indeed, the website is gone where the information is retrieved. I will retire the app
Thanks.
